docs/datamodel/links.rst

```diff
@@ -274,8 +274,30 @@ same "shirt owner" relationship is represented with a ``multi`` link.
 Under the hood, a ``multi`` link is stored in an intermediate `association
 table <https://en.wikipedia.org/wiki/Associative_entity>`_, whereas a
 ``single`` link is stored as a column in the object type where it is declared.
-As a result, single links are marginally more efficient. Generally ``single``
-links are recommended when modeling 1:N relations.
+
+Another decision often faced by EdgeDB users is whether to use a ``multi``
+link on the linking object or a backlink on the other (the linked object).
+A general rule of thumb in this case is as follows.
+
+Use a ``multi`` link if:
+
+- The relationship is relatively stable and thus not updated very frequently.
+  For example, a list of postal addresses in a user profile.
+- The number of elements in the link tends to be small.
+
+Otherwise, prefer a single link from one object coupled with a computed
+backlink on the other. This is marginally more efficient and generally
+recommended when modeling 1:N relations:
+
+.. code-block:: sdl
+
+    type Post {
+      required author: User;
+    }
+
+    type User {
+      link posts := (.<author[is Post])
+    }
```

Comment:
I might do this whole block as a `.. note::`, just to call out that it isn't really part of the "one-to-many" section. It's related to both that and the "many-to-one" section since it's advising between the two.

Also, maybe we should characterize this as the "direction" of the link. I think that's a bit clearer than the way you've characterized it here. Spitballing a rewrite of the first paragraph:
> Choosing a link direction can be tricky when dealing with these kinds of links. Should you model the link as a one-to-many using `multi`, or should you model a many-to-one (single) link and use backlinks to traverse in the other direction? A general rule of thumb in this case is as follows.

Does this approach make sense?

Oh, one more:
> Otherwise, prefer a single link from one object **type** coupled with a computed backlink on the other.